### DISCUSSION TITLE
using pg_dump() in the browser ?

### DISCUSSION THREAD

**Discussion by ludoaubert** [electric-sql/pglite#455] [Q&A] [answered]
Hi everyone, using pglite on the web browser side. I was wondering if it is possible to use pg_dump in the browser as well to produce a SQL dump of the pglite I have in memory. Thanks in advance !

**Comment by samwillis** [comment, marked as answer]
Hey @ludoaubert 

We quietly did a release of pg_dump in the browser last week, a few people have been testing it and it looking good: https://www.npmjs.com/package/@electric-sql/pglite-tools

we will probably do a patch release next week, but that will only be to improve the build process and bring it fully into CI.
